src/renderer/components/SyncView/SyncAppBar.js

```diff
@@ -7,15 +7,37 @@ import { makeStyles } from '@material-ui/core/styles'
 import { Wifi, WifiOff } from '@material-ui/icons'
 import { defineMessages, useIntl } from 'react-intl'
 import wifi from 'node-wifi'
+import { Tooltip } from '@material-ui/core'
 
 wifi.init({
   iface: null
 })
 
+const getQualityStyle = (connection) => {
+  if (connection.quality < 30) return { color: 'white', backgroundColor: 'red' }
+  if (connection.quality < 60) return { color: 'white', backgroundColor: 'orange' }
+  return { backgroundColor: '#E0E0E0' }
+}
+
+/** @returns {keyof m} */
```

Comment:
This little fella is useful for autocompletion:
<img width="1168" alt="Screenshot 2020-03-31 at 21 47 46" src="https://user-images.githubusercontent.com/1416801/78072280-22428300-739f-11ea-8a18-9245a7402ced.png">

Reply:
This does not bother me, thanks for that! What plugin is this using in VS Code?

Reply:
None, it's the default Intellisense (or whatever they call that) thingie included in VSCode. It's using JSDoc comments that [Typescript works with and understand](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#supported-jsdoc)

Reply:
That's a really great tip @thibautRe. I knew TS could support JS-Doc but I never got round to reading up how - good to know there is nothing to do!